sdk/storage/storage-queue/README.md

```diff
@@ -186,19 +187,31 @@ async function main() {
       enqueueQueueResponse.messageId
     }, service assigned request Id: ${enqueueQueueResponse.requestId}`
   );
+```
+
+### Peek a message
 
-  // Peek a message using peek method.
+`MessagesClient.peek()` allows looking at one or more messages in front of the queue. This call
+doesn't prevent other code from accessing peeked messages.
+
+```javascript
   const peekQueueResponse = await messagesClient.peek();
   console.log(
     `The peeked message is: ${
       peekQueueResponse.peekedMessageItems[0].messageText
     }`
   );
+```
+
+### Processing a message
 
-  // You de-queue a message in two steps. Call GetMessage at which point the message becomes invisible to any other code reading messages
-  // from this queue for a default period of 30 seconds. To finish removing the message from the queue, you call DeleteMessage.
-  // This two-step process ensures that if your code fails to process a message due to hardware or software failure, another instance
-  // of your code can get the same message and try again.
+Messages are processed in two steps.  First call `MessagesClient.dequeue()`. This makes the messages
+invisible to other code reading messagse from this queue for a default period of 30 seconds. When processing
+of a message is done, call `MessagesClient.delete()` with the message's `popReceipt`.
+This two-step process ensures that if your code fails to process a message due to hardware or software failure, another instance
+of your code can get the same message and try again.
```

Comment:
>This two-step process ensures that if your code fails to process a message due to hardware or software failure, another instance of your code can get the same message and try again.

Can be rephrased ?
If your code fails to process a message due to hardware or software failure, this two-step process ensures that another instance of your code can get the same message and try again.